erpnext/stock/doctype/stock_entry/stock_entry.py

```diff
@@ -1734,3 +1734,32 @@ def validate_sample_quantity(item_code, sample_quantity, qty, batch_no = None):
 			format(max_retain_qty, batch_no, item_code), alert=True)
 		sample_quantity = qty_diff
 	return sample_quantity
+
+@frappe.whitelist()
+def get_items_from_warehouse(warehouse, posting_date, posting_time, company):
+	lft, rgt = frappe.db.get_value("Warehouse", warehouse, ["lft", "rgt"])
+	items = frappe.db.sql("""
+		select i.name, i.item_name, bin.warehouse, i.item_group, i.description, i.stock_uom
+		from tabBin bin, tabItem i
+		where i.name=bin.item_code and i.disabled=0 and i.is_stock_item = 1
```

Comment:
```suggestion
		on i.name=bin.item_code where i.disabled=0 and i.is_stock_item = 1
```

Prefer explicit joins over implicit joins.